Flutter/pdf-viewer/How-to/custom-widget-on-flutterflow.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: PdfViewer Widget in FlutterFlow | Syncfusion
+description: You can learn here all about adding Syncfusion Flutter Pdf Viewer ( SfPdfViewer Widget ) widget in FlutterFlow.
```

Comment:
There are spaces at the front and back inside the bracket content. Is this valid?

<img width="349" alt="image" src="https://github.com/user-attachments/assets/90980f74-0ffb-454e-a373-389af717c0da">

Reply:
Have removed the spaces